### ISSUE TITLE
AERhr interval wrong

### ISSUE THREAD

**Issue by tommibergman** [PCMDI/cmip6-cmor-tables#211]
I am cmorizing AerChemMIP data and I found that in CMIP6_AERhr.json there is a mistake. Interval is set at 1 day while it should be 0.017361 days. Actually it looks like the interval value was added onto generic levels, at least the value is the same.

**Comment by doutriaux1** [comment]
@tommibergman thanks for spotting this. Will fix as soon as @taylor13 reviewed this issue and other tables.

**Comment by doutriaux1** [comment]
@taylor13 please advise if any other table needs touching as well. @mauzey1 we should probably add a test in cmor/prepare to check this.

**Comment by taylor13** [comment]
This isn't a problem with CMOR/PrePARe is it?  Isn't it a problem with the step where Martin's tables get translated into CMOR json files?  If so and we decide a test is needed, it should be done on the json files directly not through CMOR.  [Or perhaps I've misunderstood.]

I'll try to check the json files for frequency and then get back to you.

**Comment by doutriaux1** [comment]
@taylor13 you're right. But the tests for the CMIP6 tables are essentially the prepare tests of CMOR. So we can just add some tests to CMOR. @mauzey1 we should add these tests and see which ones fail. Then we need to see why the xml to json code picked up the wrong frequency. I can point you to the code if you don't know where it is.

**Comment by taylor13** [comment]
This needs to be corrected right away (so I can sign off on release 01.00.28).

The "approx_interval" in tables E1hr and AERhr should be "0.041667" .
The "generic_levels" in table AERhr should be "alevel alevhalf".

I don't think this information comes from the data request.  It must be provided by us (through some mechanism) when we create the json table files, so we need to fix what we provide.  Denis might be able to help.

[Also   @tommibergman :  Do you still think the approx_interval should be 0.017361 and not 0.041667?]

**Comment by tommibergman** [comment]
No, I think you are right, that the correct interval should be 0.041667.

**Comment by mauzey1** [comment]
This appears to be caused by an issue in this file in xml-cmor3-database.

https://github.com/PCMDI/xml-cmor3-database/blob/da5f42db91ef26d1e82e44cbb52b97944872032e/src/CMOR3Template.py#L144-L145
https://github.com/PCMDI/xml-cmor3-database/blob/da5f42db91ef26d1e82e44cbb52b97944872032e/src/CMOR3Template.py#L189

**Comment by mauzey1** [comment]
I also noticed in that file I posted other tables with "hr" or "hr1" in their name that have an approxInterval value of 0.017361.  Should these be changed?

**Comment by taylor13** [comment]
Please remove all the files not needed for CMIP6.  After doing that I think the only files with "hr" / "hr1" data are E1hr, E1hrClimMon, and AERhr.  

One additional change needed is that for cfsubhr and Esubhr tables please include:
"approxIntervalWarning":  "0.5",
"approxIntervalError": "0.90",
for example,
```
              "cfsubhr": { "approxInterval" : "0.017361",
                           "genericLevels"  : "alevel alevhalf",
                           "approxIntervalWarning":  "0.25",
                           "approxIntervalError": "0.75",
```

**Comment by mauzey1** [comment]
The changes for the interval values have been merged, and are now part of version 01.00.29 of the tables and CMOR 3.4.0.

Closing issue.
